sources/packages/forms/src/form-definitions/institutionwelcomepage.json

```diff
@@ -0,0 +1,3175 @@
+                                            "redrawOn": "",
+                                            "dataGridLabel": false,
+                                            "labelPosition": "top",
+                                            "description": "",
+                                            "errorLabel": "",
+                                            "autofocus": false,
+                                            "dbIndex": false,
+                                            "customDefaultValue": "",
+                                            "calculateValue": "",
+                                            "calculateServer": false,
+                                            "widget": null,
+                                            "validateOn": "change",
+                                            "validate": {
+                                                "required": false,
+                                                "custom": "",
+                                                "customPrivate": false,
+                                                "strictDateValidation": false,
+                                                "multiple": false,
+                                                "unique": false
+                                            },
+                                            "allowCalculateOverride": false,
+                                            "encrypted": false,
+                                            "showCharCount": false,
+                                            "showWordCount": false,
+                                            "allowMultipleMasks": false,
+                                            "addons": [],
+                                            "tree": false,
+                                            "lazyLoad": false,
+                                            "id": "ecvupsf"
+                                        }
+                                    ],
+                                    "width": 9,
+                                    "offset": 0,
+                                    "push": 0,
+                                    "pull": 0,
+                                    "size": "md",
+                                    "currentWidth": 9
+                                },
+                                {
+                                    "components": [
+                                        {
+                                            "title": "",
+                                            "labelWidth": "",
+                                            "labelMargin": "",
+                                            "theme": "default",
+                                            "tooltip": "",
+                                            "customClass": "formio-institution-get-started-panel",
+                                            "collapsible": false,
```

Comment:
mm..maybe we can give some meaning full, generic name for the custom class, most probably we  will need to reuse this class in part of application like a welcome page of aest or any other user

Reply:
also, Iam not seeing this custom class definition, did u forget to add it?